use qurious::error::{Error, Result};

fn main() -> Result<()> {
    Ok(())
}
